src/entries/popup/pages/home/Activity/ActivitiesList.tsx

```diff
@@ -48,9 +58,71 @@ export function Activities() {
   });
   useTransactionListForPendingTxs();
   const containerRef = useContainerRef();
+  const { currentAddress } = useCurrentAddressStore();
+  const { currentCurrency } = useCurrentCurrencyStore();
+  const { rainbowChains } = useRainbowChains();
+  const supportedMainnetIds = SUPPORTED_MAINNET_CHAINS.map((c: Chain) => c.id);
+  const { userChains } = useUserChainsStore();
+  const { isWatchingWallet } = useWallets();
+
+  const chainIds = rainbowChains
+    .filter((c) => supportedMainnetIds.includes(c.id) && userChains[c.id])
+    .map((c) => c.id);
```

Comment:
do we need the rainbowChains? can't we filter filter the supportedMainnetIds from userChains?

Reply:
hm yeah i think that would work